Library/Formula/libgraphqlparser.rb

```diff
@@ -0,0 +1,51 @@
+class Libgraphqlparser < Formula
+  desc "GraphQL query parser in C++ with C and C++ APIs"
+  homepage "https://github.com/graphql/libgraphqlparser"
+  url "https://github.com/graphql/libgraphqlparser/archive/v0.2.0.tar.gz"
+  sha256 "5064f63024c20cdc2c41970a6e9a5c7b053565db22f5f8dfb946923cb077f9de"
+
+  depends_on "cmake" => :build
+  depends_on "bison" => :recommended
+  depends_on "flex" => :recommended
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "install"
+    libexec.install "dump_json_ast"
+  end
+
+  test do
+    require "open3"
```

Comment:
nit: this looks redundant now, if you end up changing the dependencies; otherwise we can get this when we pull